### DISCUSSION TITLE
Please HELP!! with optimizing config for my AMD 3990x

### DISCUSSION THREAD

**Discussion by mechic** [swar/Swar-Chia-Plot-Manager#784] [Q&A]
Need suggestions please to maximize the most plots/day

Equipment:

SSD : SAMSUNG 980 PRO 2TB PCIe NVMe Gen4 I have 10 of these
6-single
2-raid 0
2-raid 0
(motherboard only supports max 8 m.2's, hence why I have 6 single's 2 raid 0 sets)

Mem : 256G (32g DDR4 2666 *8)

Cpu: AMD Threadripper 3990x (64C 128T)

PC is connected to a 10Gig switch to NAS

size: ?
bitfield: true
threads: ?
buckets: 128
memory_buffer: ?
max_concurrent: ?
max_concurrent_with_start_early: ?
stagger_minutes: ?
max_for_phase_1: ?
concurrency_start_early_phase: ?
concurrency_start_early_phase_delay: ?

Global Settings
 max_concurrent: ?
  max_for_phase_1: ?
  minimum_minutes_between_jobs: ?

**Comment by daiyuandian** [comment]
how many plots can you acheive this time within 1 day?

**Comment by w4d3j** [comment]
That's a lot of money spent to not know what you're doing.

What all have you tried so far?  That'd be a great place to start, since it's likely everyone's hardware and thus experience is different.

**Reply by mechic** [reply]
@daiyuandian @w4d3j What would you try with this setup?
